apptools/logger/plugin/logger_service.py

```diff
@@ -1,9 +1,11 @@
 # Standard library imports
-from cStringIO import StringIO
 import logging
 import os
 import zipfile
 
+# Third-party library imports
+from six import BytesIO
```

Comment:
Suggest using `io.BytesIO` instead of `six.BytesIO` here; then we don't need to change this when we drop Python 2 support.